data/human/human missions.txt

```diff
@@ -7455,3 +7455,141 @@ mission "Adelita 5"
+			action
+				payment 91951
+				set "mean to Howard"
+
+
+mission "Syndicate Business 2"
+	name "Purchase Bond on <planet>"
+	description "Howard from Megaparsec Incorporated is letting you buy a bond in the company for 1 million credits."
+	passengers 1
+	source "Sunracer"
+	destination "Hephaestus"
+	to offer
+		has "Syndicate Business 1: done"
+		credits > 1000000
+	on offer
+		conversation
+			`You visit the office address Howard gave you at the spaceport, where he immediately spots and heads toward you. Grateful for saving his life, he thanks you profusely and excitedly tells you what an exclusive offer you've received to be able to purchase a bond of Megaparsec Incorporated.`
+			`	"To reiterate, to purchase 1 perpetual bond of Megaparsec Incorporated I will need 1 million credits and for you to sign a few documents. Our promised returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits. Do you accept these terms?"`
+			choice
+				`	"Sure, here's the 1 million credits."`
+					goto doit
+				`	"No, I won't become an investor."`
+					decline
+			label doit
+			`After you've signed numerous dotted lines, Howard says, "Alright, we have your payment and preliminary signatures. This particular investment is in partnership with Syndicated Systems, so we'll need to submit this first set of paperwork to their office and then go to <destination> to sign the final documents and conduct a biometric scan at our secure facility. I'll go with you to help finalize the paperwork. Once that is complete, you should immediately start receiving dividend payments."`
+				accept
+	on accept
+		payment -1000000
+	on complete
+		conversation
+			`You land on <destination> and follow Howard's instructions to the biometric scan center. You hand in the first set of paperwork you filled out, and after scanning your fingerprint, retinas, and getting a blood sample, Howard comes over and says, "Excellent, only one last set of paperwork left to sign." He places one more set of documents on the table and taps the stack of papers.`
+			`	"New regulations dictate that we inform you that the proceeds of this bond may be used in high risk activities such as strip mining, clearing endangered habitats, or doing business on worlds with alleged human rights violations. All boilerplate language, of course, but you have to disclose every possibility these days." He shrugs.`
+			`	He then taps the documents the documents again and says, "This is your final decision and set of paperwork. I would also like to disclose that your initial deposit covering the full 1 million credits is non-refundable. So, as a last formality, would you like to invest in the full investment suite?"`
+			choice
+				`	"Of course."`
+					goto high
+				`	"I'm backing out, I can't ethically invest in this."`
+					decline
```

Comment:
At this point, Howard's already taken your money, so I'd expect to have at least another little bit of dialogue where he either tells you that he can't give you a refund, or does indeed give you a refund.